ceph.spec.in

```diff
@@ -94,9 +84,8 @@
 %{!?python3_pkgversion: %global python3_pkgversion 3}
 %{!?python3_version_nodots: %global python3_version_nodots 3}
 %{!?python3_version: %global python3_version 3}
-# define _python_buildid macro which will expand to the empty string when
-# building with python2
-%global _python_buildid %{?_defined_if_python2_absent:%{python3_pkgversion}}
+# define _python_buildid macro
+%global _python_buildid %{python3_pkgversion}
```

Comment:
could drop `_python_buildid` along with the comment above.

Reply:
if we drop that, what do I use instead? `3` ?

Reply:
This would be a good opportunity to look at the following macros:

```
%{!?python3_pkgversion: %global python3_pkgversion 3}
%{!?python3_version_nodots: %global python3_version_nodots 3}
%{!?python3_version: %global python3_version 3}
```

All of them default to `3` - are all of them needed? Are *any* of them needed?

Reply:
they are.

Reply:
> if we drop that, what do I use instead? 3 ?

i mean we could just use `python3_pkgversion`.

Reply:
I'll use `python3_pkgversion` and will defer to a follow-up PR to address other cleanups, to avoid pilling more changes to a rather large pull request.